UAT/V1.10.14_patchsql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE cqc."UserAudit" DROP COLUMN "EventType";
```

Comment:
You can't do this on live data - because this will remove the Event type on every single audit record already collected.

Reply:
This has now been changed to an ALTER TYPE